db/migrate/20191018171744_add_profile_photo_to_users.rb

```diff
@@ -0,0 +1,9 @@
+class AddProfilePhotoToUsers < ActiveRecord::Migration[5.2]
+  def up
```

Comment:
Doesn't require a change, but fyi that add_column is reversible, so you can do it in a `def change` instead :)